tests/e2e/blockchain/client.go

```diff
@@ -212,6 +212,18 @@ func (c *Client) RegisterCMService(
 	return nil
 }
 
+func (c *Client) RegisterCMServices(
+	ctx context.Context,
+	serviceNames []string,
```

Comment:
```suggestion
	serviceNames ...string,
```
call with inline array would look a bit cleaner, imo, without array declaration

Reply:
Done